remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/transport/triple/TripleClientInvoker.java

```diff
@@ -238,6 +214,76 @@ public void onCompleted() {
         return future;
     }
 
+    private void processSuccess(boolean needDecode, RpcInternalContext context, SofaRequest sofaRequest, Object o, SofaResponseCallback sofaResponseCallback, TripleResponseFuture future) {
+        ClassLoader oldCl = Thread.currentThread().getContextClassLoader();
```

Comment:
The current classloader is obtained here, but no switching classloader is seen below.

Reply:
fixed